sdk/python/v2beta1/setup.py

```diff
@@ -9,8 +9,61 @@
     Generated by: https://openapi-generator.tech
 """
 
+import os
+import setuptools
 
-from setuptools import setup, find_packages  # noqa: H301
+# Make sure everything is relative to setup.py
+install_path = os.path.dirname(os.path.abspath(__file__))
+os.chdir(install_path)
 
-NAME = "kubeflow"
-VERSION = "0.1"
+# Try to read description, otherwise fallback to short description
+DESCRIPTION = "Python SDK for the MPI Operator"
+try:
+    with open(os.path.join(install_path, "README.md")) as fd:
+        LONG_DESCRIPTION = fd.read()
+except Exception:
+    LONG_DESCRIPTION = DESCRIPTION
+
+def setup():
+    """
+    Main entrypoint to run setup.
+    """
+    setuptools.setup(
+        name="kubeflow-mpijob",
+        version="0.0.1",
+        author="Kubeflow Authors",
+        author_email="wg-batch@kubernetes.io",
```

Comment:
```suggestion
        author_email="kubeflow-discuss@googlegroups.com",
```

Kubernetes WG batch doesn't maintain mpi-operator. So we should set another one to here.
@terrytangyuan @alculquicondor What do you think about using `kubeflow-discuss@googlegroups.com`?

Reply:
It’s better not to spam that mailing list with bug reports

Reply:
I usually put a faux GitHub email here - the `<name>@users.noreply.github.com` but if someone has an actual email, that is likely ideal. I'm not sure how often these emails get used, but I would say if the group is protected by someone needing to join, that would get around bots that scrape the emails (and spam) and maybe even be a good approach? Someone that wants help with the package should maybe reach out to this list?

Reply:
Okay I updated to `kubeflow@users.noreply.github.com`. it is definitely useless and fake, but will satisfy the pypi requirements if you ever decide to release there (you just need the contact field defined with something resembling an email).

Reply:
No objection.

Reply:
sgtm